bin/run-browser-tests-azure

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python3
+
+# DOC: Run a subset of browser tests that we want to specifically test in Azure in CI mode.
+
+import subprocess
+
+
+def get_test_files():
+    test_files = ['src/file.test.ts']  # file upload tests
+    test_files.sort()
```

Comment:
Do you need this sort?

Reply:
Definitely not atm since it's just the one file lol. I copied it from https://github.com/civiform/civiform/blob/9ddd1479403e5f11944709fe0bec4ad23e36083a/bin/run-browser-tests-ci-batch#L35 thinking we would want a sort if we had more than one file, but since these likely won't be batched I don't think we need it. What do you think?

Reply:
Let's remove it. If we get to the point of having so many tests to run we need batch again probably means it's time to just start running everything.

Reply:
done